tracing-opentelemetry/Cargo.toml

```diff
@@ -17,28 +17,24 @@ categories = [
 keywords = ["tracing", "opentelemetry", "jaeger", "zipkin", "async"]
 license = "MIT"
 edition = "2018"
-rust-version = "1.46.0"
+rust-version = "1.56.0"
 
 [features]
 default = ["tracing-log", "metrics"]
 # Enables support for exporting OpenTelemetry metrics
 metrics = ["opentelemetry/metrics"]
 
 [dependencies]
-opentelemetry = { version = "0.17.0", default-features = false, features = ["trace"] }
+opentelemetry = { version = "0.18.0", default-features = false, features = ["trace"] }
 tracing = { path = "../tracing", version = "0.2", default-features = false, features = ["std"] }
 tracing-core = { path = "../tracing-core", version = "0.2" }
 tracing-subscriber = { path = "../tracing-subscriber", version = "0.3", default-features = false, features = ["registry", "std"] }
 tracing-log = { path = "../tracing-log", version = "0.2", default-features = false, optional = true }
 once_cell = "1.13.0"
 
-# Fix minimal-versions; opentelemetry specifies async-trait = "0.1" which breaks
-async-trait = "0.1.20"
-
 [dev-dependencies]
-async-trait = "0.1.56"
```

Comment:
Similarly, we need to put this back to fix the minimal-versions build. Then CI should pass again.